.github/workflows/rotki_ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       backend_tasks: ${{ steps.checker.outputs.backend_tasks }}
       frontend_tasks: ${{ steps.checker.outputs.frontend_tasks }}
       documentation_tasks: ${{ steps.checker.outputs.documentation_tasks }}
+      TEST_ENVIRONMENT: ${{env.TEST_ENVIRONMENT}}
```

Comment:
nit: can you make the output name lower case please for consistency with the others?

And after I am back from vacation I will also turn it into and output on the job itself.